apps/api/src/app/content-templates/usecases/parse-preview/preview-email.usecase.ts

```diff
@@ -12,7 +12,7 @@ export class PreviewEmail {
     const isEditorMode = command.contentType === 'editor';
     const [organization, content]: [OrganizationEntity, string | IEmailBlock[]] = await Promise.all([
       this.organizationRepository.findById(command.organizationId),
-      this.getContent(isEditorMode, command.content),
+      this.getContent(isEditorMode, command.content, JSON.parse(command.payload)),
```

Comment:
I would extract this to a const for reusability as you use it a bit below.

Reply:
I am assuming also this payload is validated already that is a parseable JSON so we don't have to worry to manage potential errors.